services/vision/server.go

```diff
@@ -217,6 +200,54 @@ func segmentsToProto(frame string, segs []*vision.Object) ([]*commonpb.PointClou
 	return protoSegs, nil
 }
 
+func (server *serviceServer) CaptureAllFromCamera(
+	ctx context.Context,
+	req *pb.CaptureAllFromCameraRequest,
+) (*pb.CaptureAllFromCameraResponse, error) {
+	ctx, span := trace.StartSpan(ctx, "service::vision::server::CaptureAllFromCamera")
+	defer span.End()
+	svc, err := server.coll.Resource(req.Name)
+	if err != nil {
+		return nil, err
+	}
+	capt, err := svc.CaptureAllFromCamera(ctx,
+		req.CameraName,
+		req.ReturnImage,
+		req.ReturnDetections,
+		req.ReturnClassifications,
+		req.ReturnObjectPointClouds,
+		req.Extra.AsMap(),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	objProto, err := segmentsToProto(req.CameraName, capt.PointCloudObject())
+	if err != nil {
+		return nil, err
+	}
+
+	imgProto, err := imageToProto(ctx, capt.Image(), utils.MimeTypeJPEG)
```

Comment:
don't assume JPEG here -- pass in ""

Then, in the function `imageToProto`, you can do some type testing and put in your defaults. If you have "", then check what kind of image `capt.Image()` is. If it is a Gray16 image, or a DepthMap image, then you'll want to encode it instead as a Viam Depth type `utils.MIMETypeRawDepth`. If not, check if it's a LazyEncodedImage, and put in the mime type from the lazy encoded image. 

The reason why you don't want to assume JPEG is because if the Image is a LazyEncodedImage, and the requested MIMEType encoding is not the same as what is stored, a decode/encode step will happen within `EncodeImage`. 

At the very end, if it neither a depth map, gray16, or lazy encoded image -- then you can assume JPEG